.github/workflows/python_package.yml

```diff
@@ -75,32 +75,32 @@ jobs:
         include:
           - os: macos-13
             task: regular
-            python_version: '3.11'
+            python_version: "3.11"
```

Comment:
Why were all these quoting changes necessary? If they were just something done automatically by your IDE, please revert them.

If there's a way to enforce a specific choice of quote type with `yamllint`, I'd welcome a separate PR to do that, so we don't have to think about this in the future.

Reply:
I removed these unrelated changes in https://github.com/microsoft/LightGBM/pull/6896/commits/8c9bec8a2ed164d5fda5076be6ddf42d0d018608